app/views/questions/index.html.erb

```diff
@@ -28,14 +28,34 @@
           <h4 class="panel-title">
             <div class="row valign">
               <div class="col-sm-11">
-                <!-- Toggles the modal. Data that is included: question answer, question text, and question ID -->
+                <!-- Toggles the Question modal. Data that is included: question answer, question text, and question ID -->
                 <span style="cursor:default">
                   <a data-toggle="modal" 
                     data-target="#question_display_Modal" 
                     data-answer="<%=question["answers"][0]["text"]%>" 
                     data-qid="<%=question["id"]%>" 
-                    data-question="<%= question["text"]%>">
-                      <%= question["text"]%>
+                    data-question="<%= question["text"]%>"><%= question["text"]%>
```

Comment:
I like the previous version, where the `<a>` link text was on its own line. It's clearer to me what it's doing, and that it isn't related to the `data-question` declaration on the line above it.